tap_gemini/transport.py

```diff
@@ -215,22 +229,31 @@ def request(self, method: str, url: str, *args, **kwargs) -> requests.Response:
             response.raise_for_status()
 
         # Handle HTTP errors
-        # See: https://developer.yahoo.com/nativeandsearch/guide/v1-api/error-responses.html
         except requests.HTTPError as http_error:
 
             # Clear authentication info
             if response.status_code == http.HTTPStatus.UNAUTHORIZED:
                 del self.access_token
 
-            # Log errors
-            response = http_error.response
-            self.log_response_errors(response)
+            # De-serialise JSON response
+            data = response.json()
+
+            errors = list()
+
+            try:
+                errors.extend(data['errors'])
+            except KeyError:
+                errors.append(data['error'])
```

Comment:
Why `append` here and `extend` in the other case? A clarifying comment would be nice

Reply:
I have included comments to explain why there's `append` and `extend`